backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt

```diff
@@ -115,5 +117,14 @@ class Execute : CliktCommand(help = "Starts the webserver") {
     }
 }
 
+class Migrate : CliktCommand(help = "Migrates the database") {
+    private val config by requireObject<BackendConfiguration>()
+
+    override fun run() {
+        val db = Database.setup(config)
+        runAllMigrations(db, Clock.systemDefaultZone())
```

Comment:
Maybe we should set UTC here explicitly since we use it for all our timestamps. SystemDefault can vary?